### ISSUE TITLE
💡 Paragraph typing instead of words

### ISSUE THREAD

**Issue by f88083** [kraanzu/termtyper#53]
Is it possible to type a paragraph rather than several random words?

**Comment by kraanzu** [comment]
Hey! It is
But that'd require a huge amount of paragraphs yes? Otherwise, you'll have to type in the same paragraph(s) again and again.

Any suggestions how we can tackle this? :D

**Comment by f88083** [comment]
> Hey! It is But that'd require a huge amount of paragraphs yes? Otherwise, you'll have to type in the same paragraph(s) again and again.
> 
> Any suggestions how we can tackle this? :D

chatGPT? or contribution from everyone.

**Comment by kraanzu** [comment]
Hmm..Good Ideas. I was hoping to borrow some stuff from monkeytype but it doesn't have paragraph, [only quotes](https://github.com/monkeytypegame/monkeytype/tree/master/frontend/static/quotes)

**Comment by f88083** [comment]
> Hmm..Good Ideas. I was hoping to borrow some stuff from monkeytype but it doesn't have paragraph, [only quotes](https://github.com/monkeytypegame/monkeytype/tree/master/frontend/static/quotes)

Or is it legit that just copy the content from the free ebooks?

**Comment by kraanzu** [comment]
Hmm... Not sure about that. I guess chatgpt or similar tools should be better as they'll be able to generate paragraphs based on the word count :)

**Comment by a-usr** [comment]
Alternatively we could just use MonkeyType's Quotes. Or am I missing the difference between Quotes and Paragraps?

**Comment by kraanzu** [comment]
Yes! I was considering that as well. Then maybe we can disable the punctuation/number toggles and others which won;t contribute to it

**Comment by a-usr** [comment]
> Yes! I was considering that as well. Then maybe we can disable the punctuation/number toggles and others which won;t contribute to it

Maybe even add an optional "deactivated" key or something similar for themes to use, for displaying a deactivated button / toggle (deactivated in a "you cannot use this" sense)?

**Comment by kraanzu** [comment]
Not sure about the disabled thing since the `off` state is also dimmed.
Maybe just totally remove/hide it. I'll see if I can animate it properly :)

**Comment by James4Ever0** [comment]
I would recommend typing random characters. I have tried my app for it. It works great for programmers.

https://github.com/James4Ever0/typing_training

**Comment by kraanzu** [comment]
Oh hey @James4Ever0 ! Saw you at #jobs channel :)

> I would recommend typing random characters. I have tried my app for it. It works great for programmers.

Yea funky modes could be great, I'm kinda on a break right now and will resume from mid-feb/march. Will def implement :D
